webpack.config.js

```diff
@@ -177,24 +170,19 @@ const webpackConfig = {
 				return outputPath;
 			},
 		} ),
-		new WebpackRTLPlugin( {
-			minify: {
-				safe: true,
-			},
-		} ),
+		new RtlCssPlugin( '[name].rtl.css' ),
```

Comment:
This is something I need to test a bit more thoroughly, I've moved to another rtl css plugin that is compliant with webpack 5 and works alongside MiniCssExtractPlugin.